elide-core/src/main/java/com/yahoo/elide/core/datastore/inmemory/HashMapDataStore.java

```diff
@@ -42,7 +46,20 @@ public HashMapDataStore(Set<Package> beanPackages) {
         for (Package beanPackage : beanPackages) {
             ClassScanner.getAnnotatedClasses(beanPackage, Include.class).stream()
                 .filter(modelClass -> modelClass.getName().startsWith(beanPackage.getName()))
-                .forEach(modelClass -> dataStore.put(modelClass, Collections.synchronizedMap(new LinkedHashMap<>())));
+                .filter(modelClass -> dictionary.getFirstAnnotation(modelClass,
+                        Arrays.asList(Include.class, Exclude.class)) instanceof Include)
+                .forEach(modelClass -> dataStore.put(modelClass,
+                        Collections.synchronizedMap(new LinkedHashMap<>())));
+        }
+
+        for (Package beanPackage : beanPackages) {
```

Comment:
What's the purpose of the second scan?  The classpath scans are pretty expensive (they take multiple seconds).  This could slow down the build quite a bit.

Reply:
Used one scan.